client/src/Components/Home/Home.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from "react";
+import Categories from "../Categories/Categories";
+import Slider from "../Carousel/Slider";
+import "./Home.scss";
+import { connect } from "react-redux";
+import { homeAction } from "../../_actions";
+
+const Home = (props) => {
+
+  useEffect(() => {
+    props.getCategories();
+    props.getBanners();
+  }, []);
+
+  const { banners, categories } = props;
+
+  return (
+    <main className="home-content">
+      { banners.length ? <Slider slides={banners} /> : ""}
+      { categories.length ? <Categories categoryList={categories} /> : ""}
+    </main>
```

Comment:
exclude slider from main. Wrap Categories only in main

Reply:
changes done